sound/soc/sof/sof-priv.h

```diff
@@ -477,10 +598,27 @@ int snd_sof_get_status(struct snd_sof_dev *sdev, u32 panic_code,
 		       size_t stack_size);
 int snd_sof_init_trace_ipc(struct snd_sof_dev *sdev);
 
+/*
+ * VirtIO
+ */
+#define GUEST_COMP_ID_OFFSET  0x100
+int sof_virtio_submit_guest_ipc(struct snd_sof_dev *sdev, int vm_id,
```

Comment:
if these are private APIs to only be used within SOF then we can prefix them as sof_virtio_